docs/user_manual/site_configuration.md

```diff
@@ -0,0 +1,191 @@
+
+Some institutes will have a specific and uniform way of naming things. MISO allows you to customize the validation of
+some fields in order to enforce these naming schemes:
+
+* All item names
+* Sample alias
+* Library alias
+* Project ShortName
+
+It may also be possible to auto-generate aliases for your samples and/or libraries.
+
+Both validation and alias generation must be configured by your MISO administrators.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Taxon Lookup" %}
+
+A [NCBI](https://www.ncbi.nlm.nih.gov/) service may be used to validate the scientific name entered for samples to
+ensure that the name is entered correctly. If this is enabled, you will not be able to save a sample until a valid
+scientific name is entered. You can browse the taxonomy database using NCBI's
+[Taxonomy Browser](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi).
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Automatic Barcode Generation" %}
+
+If automatic barcode generation is enabled for your site, MISO will automatically generate a unique barcode any time
+you leave the barcode field blank for an item. These barcodes, whether auto-generated or not, may be printed from MISO
+to a label printer. See the [Barcode Label Printers section](barcode_label_printers.html) for more information.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Barcode Scanners" %}
+
+Barcode scanners may be used to enter barcodes into any text-entry area in MISO. Most barcode scanners allow you to
+configure a key to send after scanning. If you configure your barcode scanners to send the Enter key after a scan, that
+will save you from having to press Enter or click on a button or the next item every time to scan a barcode. This
+configuration is done on the barcode scanner itself, and you should see the scanner's manual for configuration
+instructions.
+
+It is also possible to use
+[Thermo Fisher VisionMate™ High Speed Barcode Readers](https://www.thermofisher.com/order/catalog/product/312800) to
+scan entire boxes of tubes at once. These bulk scanners must be configured by your MISO administrators to work in MISO.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Report Links" %}
+
+{% assign figure = figure | plus: 1 %}
+If you are generating reports for projects or runs outside of MISO, it may be possible to automatically link to them
+from the MISO Project and Run pages. For projects, it must be possible to generate the report's URL given the project's
+ID, name, and short name. For runs, it must be possible to generate the report's URL given the run's ID, name, and
```

Comment:
Project: `ID, name, or short name`
Run: `ID, name, or alias`

Reply:
Then maybe `and/or`? `and` alone made it sound to my ears like you might need to have all of them in the report's URL. But maybe that's just me?

Reply:
and/or

Reply:
Changed the wording to "using x, y, and/or z" and added examples